proto/storage/image_integration.proto

```diff
@@ -35,6 +35,13 @@ message ImageIntegration {
     bool   skip_test_integration = 18;
 
     reserved 19; // Previously, scannerv2
+
+    message Source {
+        string cluster_id = 1;
+        string namespace  = 2;
+        string name       = 3;
```

Comment:
Call this `secret_name` or `image_pull_secret_name`? It's a bit unclear otherwise